bin/celeste.jl

```diff
@@ -32,14 +38,19 @@ The `score-field` subcommand is not yet implemented for the new API.
 function main()
     args = docopt(DOC, version=v"0.1.0", options_first=true)
     Celeste.set_logging_level(args["--logging"])
-    if args["infer-box"]
+    if args["stage-box"] || args["infer-box"]
         ramin = parse(Float64, args["<ramin>"])
         ramax = parse(Float64, args["<ramax>"])
         decmin = parse(Float64, args["<decmin>"])
         decmax = parse(Float64, args["<decmax>"])
-        outdir = args["<outdir>"]
-        Celeste.infer_box_nersc(ramin, ramax, decmin, decmax, outdir)
-    else
+        if args["stage-box"]
+            Celeste.stage_box_nersc(ramin, ramax, decmin, decmax)
+        elseif args["infer-box"]
+            outdir = args["<outdir>"]
+            Celeste.infer_box_nersc(ramin, ramax, decmin, decmax, outdir,
+                                    stage=args["--stage"])
+        end
+    elseif args["infer-field"]
```

Comment:
Changing `else` to `elseif` here seems to prevent us from calling subcommands like `infer-object` and`score-object`.